RULES_DESCRIPTIONS.md

```diff
@@ -432,7 +432,7 @@ Example:
 
 ```toml
 [imports-blacklist]
-  arguments =["crypto/md5", "crypto/sha1"]
+  arguments =["crypto/md5", "crypto/sha1", "crypto/**/pkix"]
```

Comment:
You could also complete the documentation of the configuration by explaining that regexps are also accepted